### DISCUSSION TITLE
How to run a makefile in Windows?

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#24] [Q&A] [answered]
As the question describes

**Comment by andatoshiki** [comment, marked as answer]
Check out [[GnuWin](http://gnuwin32.sourceforge.net/)](http://gnuwin32.sourceforge.net/)'s [make *(for windows)*](http://gnuwin32.sourceforge.net/packages/make.htm), which provides a native port for Windows (without requiring a full runtime environment like Cygwin)

If you have [[winget](https://github.com/microsoft/winget-cli)](https://github.com/microsoft/winget-cli), you can install via the CLI like this:

```sh
winget install GnuWin32.Make
```

Also, be sure to add the install path to your system `PATH`:

```none
C:\Program Files (x86)\GnuWin32\bin
```
